keyboards/riot_pad/info.json

```
@@ -0,0 +1,55 @@
+{
+    "manufacturer": "ShandonCodes",
+    "keyboard_name": "RiotPad",
+    "maintainer": "ShandonCodes",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B14", "A8", "A9"],
+        "rows": ["B13", "B15"]
+    },
+    "ws2812": {
+        "pin": "A7",
+        "driver": "spi"
+    },
+    "rgblight": {
+        "led_count": 12,
+        "animations": {
+            "breathing":true,
+            "alternating":true,
+            "christmas":true,
+            "knight":true,
+            "rainbow_mood":true,
+            "rainbow_swirl":true,
+            "snake":true,
+            "static_gradient":true,
+            "twinkle":true
+        }
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1000",
+        "vid": "0x7877"
+    },
+    "layouts": {
+        "LAYOUT_ortho_2x3": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 2], "x": 2, "y": 1}
+            ]
+        }
+    }
+}
```

1. add community layout
2. EoF added
```suggestion
    },
    "community_layouts": ["ortho_2x3"],
    "layouts": {
        "LAYOUT_ortho_2x3": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2, "y": 1}
            ]
        }
    }
}

```